webapp/src/components/modals/channel_settings/edit_channel_settings.tsx

```diff
@@ -141,6 +149,46 @@ export default class EditChannelSettings extends PureComponent<Props, State> {
         const filters = {...this.state.filters};
         filters[id] = finalValue;
         this.setState({filters});
+        this.clearConflictingErrorMessage();
+    };
+
+    clearConflictingErrorMessage = () => {
+        this.setState({conflictingError: null});
+    }
+
+    handleIssueChange = (id: keyof ChannelSubscriptionFilters, value: string[]) => {
+        let finalValue = value;
+        if (!finalValue) {
+            finalValue = [];
+        } else if (!Array.isArray(finalValue)) {
+            finalValue = [finalValue];
+        }
+        const filters = {...this.state.filters};
+        filters[id] = finalValue;
```

Comment:
Since we know `issue_types` is using a select with `isMulti: true`, this first block can be shorted to:
```suggestion
		handleIssueChange = (id: keyof ChannelSubscriptionFilters, value: string[] | null) => { // react-select will pass null if no issue_types are selected
		const finalValue = value || [];
        const filters = {...this.state.filters, issue_types: finalValue};
```